src/index.ts

```diff
@@ -24,6 +24,8 @@ export type { State } from './copier';
 const { isArray } = Array;
 const { assign, getPrototypeOf } = Object;
 
+const _getPrototypeOf = getPrototypeOf || (obj) => obj.__proto__
```

Comment:
Both for smaller footprint, but also to avoid the leading `_` prefix bailing out ESLint, can you update to instead do property access and avoid the local ref above?

```ts
const { assign } = Object;

const getPrototypeOf = Object.getPrototypeOf || (object) => object.__proto__;
```

Reply:
Will do, sorry, I didn't see the GH notification until today.

Reply:
Done. I kept the `obj` variable name for the lambda function, to minimize confusion with `Object`.